tests/engine/daemon/test_client.py

```
@@ -393,3 +393,60 @@ def test_restart_daemon_missing_version_file_configuration(stopped_daemon_client
+    """Test that with ``wait=True`` a call that outlasts the timeout raises, since the response is only sent once the
+    daemon acted on the command."""
+    from circus.exc import CallError
+
+    with (
+        patch.object(DaemonClient, 'get_daemon_pid', return_value=1),
+        patch.object(DaemonClient, '_is_pid_file_stale', False),
+        patch.object(DaemonClient, 'get_client') as get_client,
+    ):
+        get_client.return_value.__enter__.return_value.call.side_effect = CallError('Timed out.')
+
+        with pytest.raises(DaemonTimeoutException, match='Connection to the daemon timed out.'):
```

Ruff (0.16.0)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)